lms/djangoapps/instructor/views/api_urls.py

```diff
@@ -140,4 +140,13 @@
     url(r'^start_certificate_generation',
         'instructor.views.api.start_certificate_generation',
         name='start_certificate_generation'),
+
+    url(r'^create_certificate_exception/(?P<white_list_student>[^/]*)',
```

Comment:
I think the above comment can be deleted because the 'other' urlpattern was removed.  Also this comment can be deleted after the above comment is deleted :)